src/app/app.component.ts

```diff
@@ -110,8 +114,13 @@ export class AppComponent {
     alert(JSON.stringify(row));
   }
 
-  logAll(data: any) {
-    console.log('table triggered event');
+  logAll(data: IPagingInput) {
```

Comment:
method `logAll` is used in many argument format, not only used with `IPagingInput`